e2e/cypress/e2e/conferences.test.js

```diff
@@ -2,6 +2,16 @@ import { onlyOn } from '@cypress/skip-test';
 import moment from 'moment';
 
 describe('Conference Search', () => {
+  it('has search results', () => {
+    cy.registerRoute();
+    cy.visit('/conferences?start_date=all');
```

Comment:
is there any test that uses the default parameter for start_date `start_date=upcoming`?

Reply:
No, there's not, I will add the test